src/node_buffer.cc

```diff
@@ -623,7 +623,8 @@ void StringWrite(const FunctionCallbackInfo<Value>& args) {
   Local<String> str = args[0]->ToString(env->isolate());
 
   if (encoding == HEX && str->Length() % 2 != 0)
-    return env->ThrowTypeError("Invalid hex string");
+    return env->ThrowTypeError("Hex strings must have an even number of "
```

Comment:
@bnoordhuis does our C++ style allow strings split like this (the linter doesn't complain)?